### PR TITLE
Add RBAC v1, Integrations v1 client tests

### PR DESCRIPTION
**Pull request by catastrophe-brandon**
* Adds coverage for RBAC v1 API for groups, roles, principals, and access
* Introduces the ability to run both RBAC v1 and v2 client tests in the same pipeline.
* Introduces some baseline Integrations API v1 tests.

### CODE REVIEW

File: packages/integrations/tsconfig.integration.v1.spec.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "compilerOptions": {
+    "outDir": "../../dist/out-tsc",
+    "module": "commonjs",
+    "types": ["jest", "node"]
+  },
+  "include": [
+    "jest.config.ts",
+    "packages/integrations/tests/integration/integrations.v1.integration.test.ts"
+  ],
+  // explicitly exclude the workspaces test because jest insists on running it for some reason
```
Comment (apinkert) [inline packages/integrations/tsconfig.integration.v1.spec.json:12]: Do we still need these comments?

File: packages/rbac/tsconfig.integration.v1.spec.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "compilerOptions": {
+    "outDir": "../../dist/out-tsc",
+    "module": "commonjs",
+    "types": ["jest", "node"]
+  },
+  "include": [
+    "jest.config.ts",
+    "packages/rbac/tests/rbac.v1.test.ts"
+  ],
+  // explicitly exclude the workspaces test because jest insists on running it for some reason
```
Comment (apinkert) [inline packages/rbac/tsconfig.integration.v1.spec.json:12]: Same here, is the exclude actually needed here?

**Review by apinkert — APPROVED** [review]
This looks good to me overall. One small thing, is it possible to remove the `npm run build:no-cache` from the first part of the `test:integration` script in the `package.json`? Running a build for every client without caching prior to integration tests for RBAC and Integrations seems unnecessary

**Comment by catastrophe-brandon** [comment]
IIRC, I think the reason we included `build:no-cache` was because we were seeing some cached/outdated client data while executing the pipeline. Possibly cached/residual data lingering on the runner image? @florkbr any reservations about removing this?

**Pull request merged**